plugins/nv-filters/data/locale/en-US.ini

```diff
@@ -0,0 +1,16 @@
+NvFilters.SuppressLevel="Suppression Level"
+Nvafx.Intensity="Suppression Intensity"
```

Comment:
If we're using `Nvafx` as the prefix for Audio Filters, then `SuppressLevel` should use `Nvafx`. If we're using `NvFilters` as the prefix for all/most strings, then we should just use that.

Reply:
agreed, i've prefixed now all the strings with Nvafx for audio and Nvvfx for video.